workflows/subwf-rna-starsolo.wdl

```diff
@@ -66,13 +67,15 @@ workflow wf_rna {
            dups_log = qc_rna.rna_duplicates_log
     }
 
-    call share_task_seurat.seurat as seurat {
-        input:
-            rna_matrix = generate_h5.h5_matrix,
-            genome_name = genome_name,
-            umap_dim = umap_dim,
-            umap_resolution = umap_resolution,
-            prefix = prefix
+    if (count_only == false) {
```

Comment:
sorry I am going to be annoying about this, could you change it to 
if (  !count_only )
ty